source/common/http/path_utility.cc

```diff
@@ -92,13 +82,14 @@ PathUtil::UnescapeSlashesResult PathUtil::unescapeSlashes(RequestHeaderMap& head
   }
   const absl::string_view query = absl::ClippedSubstr(original_path, query_start);
 
-  // TODO(yanavlasov): optimize this by adding case insensitive matcher
   std::string decoded_path{path};
-  unescapeInPath(decoded_path, "%2F", "/");
-  unescapeInPath(decoded_path, "%2f", "/");
-  unescapeInPath(decoded_path, "%5C", "\\");
-  unescapeInPath(decoded_path, "%5c", "\\");
-  headers.setPath(absl::StrCat(decoded_path, query));
+  std::map<const char*, const char*> replacements;
```

Comment:
```suggestion
  static const std::vector<std::pair<absl::string_view, absl::string_view>> replacements{{"2F", "/"}, {"2f", "/"}, ...};
```

And then remove dynamic initialization below.

Reply:
At first, it was written like this, but the code format verification did not pass

Reply:
Ok